styles/styles.css

```diff
@@ -66,6 +66,7 @@
     --heading-font-size-m: 22.56px;
     --heading-font-size-s: 16px;
     --heading-font-size-xs: 11.36px;
+    --nav-height: 75px;
```

Comment:
Not sure we need this property in this `@media` rules. We already have it in the main `:root` rule.